lib/gettext.js

```diff
@@ -7,7 +7,7 @@
     // default values that could be overriden in i18n() construct
     var defaults = {
       domain: 'messages',
-      locale: document.documentElement.getAttribute('lang') || 'en',
+      locale: window && document.documentElement.getAttribute('lang') || 'en',
```

Comment:
Operators precedence is ok :
```
c2c@vbox-perso:gettext.js $ nvm use 7
Now using node v7.3.0 (npm v3.10.10)
c2c@vbox-perso:gettext.js $ node
> true && 'test' || 'blah'
'test'
> undefined && 'test' || 'blah'
'blah'
c2c@vbox-perso:gettext.js $ nvm use 0.12 
Now using node v0.12.18 (npm v2.15.11)
c2c@vbox-perso:gettext.js $ node
> true && 'test' || 'blah'
'test'
> undefined && 'test' || 'blah'
'blah'
```

But I can change it for readability if you prefer